ci-operator/config/Azure/ARO-HCP/Azure-ARO-HCP-main__periodic.yaml

```diff
@@ -69,57 +107,87 @@ tests:
     - ref: aro-hcp-provision-generate-resource-group-name
     test:
     - chain: aro-hcp-provision-create-functional-cluster
-- as: integration-e2e-parallel
+- as: prod-uksouth-e2e-parallel
   cron: 0 */4 * * *
   steps:
     env:
-      ARO_HCP_SUITE_NAME: integration/parallel
-      CUSTOMER_SUBSCRIPTION: ARO SRE Team - INT (EA Subscription 3)
+      ARO_HCP_SUITE_NAME: prod/parallel
+      CUSTOMER_SUBSCRIPTION: ARO Hosted Control Planes (EA Subscription 1)
       LOCATION: uksouth
     post:
     - ref: aro-hcp-provision-aro-hcp-gather-extra
     - chain: aro-hcp-provision-teardown-cluster
     test:
     - ref: aro-hcp-tests-run-aro-hcp-tests
-- as: stage-e2e-parallel
+- as: delete-expired-prod-uksouth-resource-groups
+  cron: 7,37 * * * *
+  steps:
+    env:
+      CUSTOMER_SUBSCRIPTION: ARO Hosted Control Planes (EA Subscription 1)
+    test:
+    - ref: aro-hcp-provision-delete-all-expired-resource-groups
+- as: create-aro-hcp-in-prod-brazilsouth
```

Comment:
remove this job.  For every extended region, the e2e-parallel obviates the need for this separate check.